.env

```diff
@@ -1,5 +1,5 @@
 NODE_ENV=development
 VITE_SEQUENCESCAPE_BASE_URL=http://localhost:3000/api/v2
 VITE_SEQUENCESCAPE_API_KEY=development
-VITE_PRINT_MY_BARCODE_BASE_URL=http://localhost:3000
-VITE_LABEL_TEMPLATE_ID=33
+VITE_PRINT_MY_BARCODE_BASE_URL=http://localhost:9292
+VITE_LABEL_TEMPLATE_NAME=sqsc_96plate_label_template_code128
```

Comment:
Do we need this aynmore? The label template name is the same across instances.

Reply:
Yeah good point, I will move it to the code directly.